ISISPostProcessRPM/rpmbuild/autoreduce-mq/usr/bin/AutoreductionProcessor/queue_processor.py

```diff
@@ -23,10 +23,10 @@ def on_message(self, headers, data):
         destination = headers['destination']
         logger.debug("Received frame destination: " + destination)
         logger.debug("Recieved frame priority: " + headers["priority"])
-
+	
```

Comment:
Can tabs be replaced with white spaces please. We done currently check this code against the PEP 8 standard but from memory they are not recommended. Anyhow I don't like them since you risk getting different view of indentations in different editor because different editor view tabs differently